### ISSUE TITLE
`get_module_source` exception in Sphinx 1.8.0 when using autodoc on modules importing eggs

### ISSUE THREAD

**Issue by tkhyn** [sphinx-doc/sphinx#5437]
Hi, I ran into an issue when trying to deploy our documentation on a Win10 CI server, using the recent Sphinx 1.8.0 release.

### Problem

`get_module_source` raises a `ValueError` for some packages installed as `.egg`.
See https://github.com/tkhyn/sphinx180-automodule-bug

#### Procedure to reproduce the problem

On Windows 7 or 10, Python 3.6.5 as the default python interpreter and from a fresh clean virtual environment:

```
git clone git@github.com:tkhyn/sphinx180-automodule-bug.git
cd sphinx180-automodule-bug
python bootstrap.py
bin\buildout
bin\sphinx-apidoc -o doc\_apidoc autodoc_bug
bin\sphinx-build doc doc\_build
```

#### Error logs / results
```
D:\dev\.env\venv\buildout\Scripts\python.exe D:/dev/sandbox/python/sphinx-1.8.0_bug/bin/sphinx-build-script.py doc doc\_build
Running Sphinx v1.8.0
loading pickled environment... done
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 0 source files that are out of date
updating environment: [] 0 added, 1 changed, 0 removed
reading sources... [100%] _apidoc/sphinx180_bug
Exception occurred:
  File "d:\dev\.env\buildout\eggs\sphinx-1.8.0-py3.6.egg\sphinx\util\__init__.py", line 318, in get_module_source
    eggpath, _ = re.split('(?<=\\.egg)/', filename)
ValueError: not enough values to unpack (expected 2, got 1)
The full traceback has been saved in C:\Users\Thomas\AppData\Local\Temp\sphinx-err-o4loxs60.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!

Process finished with exit code 2
```

#### Expected results
As with Sphinx 1.7.8, no Exception raised and documentation generated normally

### Reproducible project / your project
See https://github.com/tkhyn/sphinx180-automodule-bug

### Environment info
- OS: Win7, Win10, could not reproduce on Linux
- Python version: 3.6.5
- Sphinx version: 1.8.0

**Comment by tk0miya** [comment]
Sorry for inconvenience. I just made #5442 to fix this.
Please try it if you have time.

Thanks,

**Comment by tk0miya** [comment]
Fixed by #5442.
Thank you for reporting
